tests/gis_tests/geos_tests/test_geos.py

```diff
@@ -726,6 +727,31 @@ def _test_buffer(self, geometries, buffer_method_name):
                     self.assertAlmostEqual(exp_ring[k][0], buf_ring[k][0], 9)
                     self.assertAlmostEqual(exp_ring[k][1], buf_ring[k][1], 9)
 
+    def _test_is_ccw(self):
+        lr = LinearRing((0, 0), (1, 0), (0, 1), (0, 0))
+        self.assertIs(lr.is_ccw, True)
+        lr.reverse()
+        self.assertIs(lr.is_ccw, False)
+
+        with self.assertRaisesMessage(ValueError, 'Orientation of empty LinearRing cannot be determined.'):
+            LinearRing().is_ccw
+
+    @unittest.skipIf(geos_version_tuple() < (3, 7), 'GEOS >= 3.7.0 is required')
+    def test_is_ccw_geos(self):
+        self._test_is_ccw()
+
+    @mock.patch('django.contrib.gis.geos.libgeos.geos_version', lambda: b'3.7.0')
```

Comment:
I think that this can be replaced with the following as above:
```python
@unittest.skipIf(geos_version_tuple() < (3, 7), 'GEOS >= 3.7.0 is required')
```

I think we only need to worry about testing the GEOS behaviour if it is actually supported; even if this is only testing the exception-raising behaviour.